flexiblepricing/management/commands/configure_tiers.py

```diff
@@ -0,0 +1,283 @@
+            nargs="?",
+            default=self.PROGRAM_TITLE,
+        )
+
+        parser.add_argument(
+            "--program-abbrev",
+            type=str,
+            help=f"Program abbreviation (to use in discount code names, default {self.PROGRAM_ABBREV})",
+            nargs="?",
+            default=self.PROGRAM_ABBREV,
+        )
+
+        parser.add_argument(
+            "--tier-info",
+            type=FileType(mode="r"),
+            help=f"Tiers to create (in CSV format: threshold,discount type,discount amount)",
+        )
+
+    def handle(self, *args, **kwargs):  # pylint: disable=unused-argument
+        # Set defaults
+
+        readable_id = (
+            kwargs["program"] if "program" in kwargs else self.PROGRAM_READABLE_ID
+        )
+        program_title = (
+            kwargs["program_name"] if "program_name" in kwargs else self.PROGRAM_TITLE
+        )
+        program_abbrev = (
+            kwargs["program_abbrev"]
+            if "program_abbrev" in kwargs
+            else self.PROGRAM_ABBREV
+        )
+
+        current_year = date.today().year
+        last_year = datetime(
+            current_year - 1, 1, 1, tzinfo=pytz.timezone(settings.TIME_ZONE)
+        )
+        discounts_and_tiers = [
+            {
+                "tier": {"threshold": 0},
+                "discount": {
+                    "discount_code": f"{program_abbrev}-fa-tier1-{current_year}",
+                    "discount_type": "dollars-off",
+                    "amount": 750,
+                },
+            },
+            {
+                "tier": {"threshold": 25000},
+                "discount": {
+                    "discount_code": f"{program_abbrev}-fa-tier2-{current_year}",
+                    "discount_type": "dollars-off",
+                    "amount": 650,
+                },
+            },
+            {
+                "tier": {"threshold": 50000},
+                "discount": {
+                    "discount_code": f"{program_abbrev}-fa-tier3-{current_year}",
+                    "discount_type": "dollars-off",
+                    "amount": 500,
+                },
+            },
+            {
+                "tier": {"threshold": 75000},
+                "discount": {
+                    "discount_code": f"{program_abbrev}-fa-tier4-{current_year}",
+                    "discount_type": "percent-off",
+                    "amount": 0,
+                },
+            },
+        ]
+        content_type = ContentType.objects.filter(
+            app_label="courses", model="program"
+        ).first()
+
+        # Step zero: if the user supplied a defaults file, then parse that
+
+        if "tier_info" in kwargs and kwargs["tier_info"] is not None:
+            discounts_and_tiers = []
+            found_zero_tier = False
+
+            self.stdout.write("Reading tier info from file...")
+
+            with kwargs["tier_info"] as tierfile:
+                csvreader = csv.reader(tierfile)
```

Comment:
For better readability of code, We could also use csv.DictReader
```suggestion
                csvreader = csv.DictReader(tierfile)
```